mongomock/aggregate.py

```diff
@@ -369,7 +372,12 @@ def _accumulate_group(output_fields, group_list):
             continue
         for operator, key in six.iteritems(value):
             key_getter = functools.partial(_parse_expression, key)
```

Comment:
you don't need the getter anymore: please inline it `values.append(_parse_expression(key, doc)`